src/roadmap/release-checklist.md

```diff
@@ -210,16 +212,7 @@ the previous tag that was generated in the repository.
 deisrel changelog individual $REPO_NAME $PREVIOUS_TAG $CURRENT_SHA $WORKFLOW_RELEASE
 ```
 
-This command will output the new changelog entry to STDOUT. Copy it and prepend it to the existing `CHANGELOG.md` file.
-
-Finally, commit, push and submit a pull request for your changes:
-
-```console
-git commit CHANGELOG.md -m "doc(CHANGELOG.md): add entry for $WORKFLOW_RELEASE"
-git push -u $YOUR_FORK_REMOTE release-$WORKFLOW_RELEASE_SHORT
-```
-
-Before you continue, ensure pull requests in all applicable repositories are reviewed, and merge them.
+This command will output the new changelog entry to STDOUT. Copy it and paste into the GitHub release notes section for the release tag.
```

Comment:
we should probably indicate that the command run/copy/paste dance needs to be done for every repo